app/scripts/directives/podDonut.js

```diff
@@ -49,7 +49,7 @@ angular.module('openshiftConsole')
           type: "donut",
           bindto: '#' + $scope.chartId,
           donut: {
-            expand: false,
+            expand: true,
```

Comment:
This was disabled when we added empty donuts when there are no pods. Can you double check this case?

https://github.com/openshift/origin/pull/5464#discussion-diff-43297113